src/main/java/com/cinema/controller/CinemaHallController.java

```diff
@@ -0,0 +1,38 @@
+package com.cinema.controller;
+
+import com.cinema.model.CinemaHall;
+import com.cinema.model.dto.hall.CinemaHallRequestDto;
+import com.cinema.model.dto.hall.CinemaHallResponseDto;
+import com.cinema.service.CinemaHallService;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.modelmapper.ModelMapper;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("cinema-halls")
```

Comment:
```suggestion
@RequestMapping("/cinema-halls")
```
let's add this everywhere

Reply:
done